README.md

```diff
@@ -1 +1,63 @@
-# Global-Summit
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# TPPS Conference (Microverse captsone 1)
+
+> This website (purely fictional) is 
+- Homepage             
+![](images/capstone-desktop.png.png)
+- About Page
+![](images/capstone-about-desktop.png)
+- Mobile Homepage
+![](images/capstone-mobile.png)
+- Mobile About Page
+![](images/capstone-about-mobile.png)
+
```

Comment:
- [ ] [OPTIONAL] I can say that It is better to add screenshots in your README file so as to give visitors a quick preview of what your site looks like but It doesn't look professional to add a lot of screenshots because they cover up large space. I would suggest you stick with not more than two screenshots here in your README